test/unit/Hits.test.js

```diff
@@ -169,57 +173,59 @@ describe('Hits', () => {
     describe('Return 200 with data', () => {
       let component;
       let spyUpdateSearchKeywords;
+      let spyUpdateSearchResults;
       let spyUpdateFacets;
       let spyUpdatePage;
-      let spyUpdateSearchResults;
       let spyAxios;
 
       before(() => {
+        spyAxios = sinon.spy(axios, 'get');
         mock
           .onGet('/api?q= owner:"orgs:1000"')
           .reply(200, response);
 
-        spyAxios = sinon.spy(axios, 'get');
         spyUpdateSearchKeywords = sinon.spy(Actions, 'updateSearchKeywords');
+        spyUpdateSearchResults = sinon.spy(Actions, 'updateSearchResults');
         spyUpdateFacets = sinon.spy(Actions, 'updateFacets');
         spyUpdatePage = sinon.spy(Actions, 'updatePage');
-        spyUpdateSearchResults = sinon.spy(Actions, 'updateSearchResults');
 
         component = mount(<Hits query="fire" facets={facets.single} hits={2} />, {
           context: { router: [] },
         });
       });
 
       after(() => {
-        spyAxios.restore();
-        spyUpdateSearchKeywords.restore();
-        spyUpdateFacets.restore();
-        spyUpdatePage.restore();
-        spyUpdateSearchResults.restore();
+        axios.get.restore();
+        Actions.updateSearchKeywords.restore();
+        Actions.updateSearchResults.restore();
+        Actions.updateFacets.restore();
+        Actions.updatePage.restore();
         mock.reset();
       });
 
       it('should be clicked and Action called', () => {
         component.find('.removeKeyword').simulate('click');
 
-        expect(spyAxios.calledOnce).to.be.true;
-        expect(spyAxios.calledWith('/api?q= owner:"orgs:1000"')).to.be.true;
+        expect(spyAxios.callCount).to.equal(1);
+        expect(spyAxios.calledWithExactly('/api?q= owner:"orgs:1000"')).to.be.true;
 
-        // expect(spyUpdateSearchKeywords.calledOnce).to.be.true;
-        // expect(spyUpdateSearchKeywords.calledWith('')).to.be.true;
+        setTimeout(() => {
```

Comment:
It seems that the `setTimeout` helps and setting the time to `0` or `10` is fine. You just *can't* re-run them more than once if you're running the mocha tests with the `watch` flag turned on.

It's also much easier to verify that these tests return the correct result if done individually.

CC @ktp242 @holingpoon